middleware/proxy/upstream.go

```diff
@@ -166,6 +169,12 @@ func parseBlock(c *parse.Dispenser, u *staticUpstream) error {
 			return c.ArgErr()
 		}
 		u.proxyHeaders.Add(header, value)
+	case "hide_header":
+		var header, value string
+		if !c.Args(&header, &value) {
+			return c.ArgErr()
+		}
+		u.hiddenHeaders.Add(header, value)
```

Comment:
There's no purpose. It was a simple copy/past from the proxy_header case above and I've forgotten to edit it. Sorry :smile_cat: 

If you have any other question, just tell me.